dist/origin-web-common-ui.js

```diff
@@ -1386,11 +1388,21 @@ angular.module('openshiftCommonUI')
     };
   })
   .filter('imageStreamTagIconClass', function(imageStreamTagAnnotationFilter) {
-  return function(resource, /* optional */ tagName) {
-    var icon = imageStreamTagAnnotationFilter(resource, "iconClass", tagName);
-    return (icon) ? icon : "fa fa-cube";
-  };
-});
+    return function(resource, /* optional */ tagName) {
```

Comment:
All I'm using this for is https://github.com/openshift/origin-web-console/pull/2708/files#diff-04a314c1b6d2d3fc082233f2ee89fa0cR106, reading, not writing.

Reply:
@spadgett ping, objections?  I'm not setting anything, this is just used to get at the annotations.  I could inline a similar fn, but figured this is where it should live.